src/pages/About.tsx

```
@@ -0,0 +1,107 @@
+      <p className="text-[#fff] text-sm md:text-base">
+        A talented team of Cryptocurrency experts based in London
+      </p>
+      <hr className="w-[50px] border border-[#fa9e1f]" />
+    </div>
+    <div className="grid grid-cols-1 sm:grid-cols-2 md:grid-cols-4 gap-5 bg-black py-8">
+      {experts &&
+        experts.map((expert,index) => (
+          <div key={index}>
+            <ExpertCard expert={expert} />
+          </div>
+        ))}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)